### ISSUE TITLE
ps_categorytree unable to hide root category from 3.0.0

### ISSUE THREAD

**Issue by wwatu1337** [PrestaShop/PrestaShop#37617]
### Prerequisites

- [x] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [x] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

ps_categorytree 3.0.0 introduced a new logic where no categories would be displayed if root category was hidden.

Previously it was possible to hide the root category view that shows all the products and only have subcategories displayed as "root" categories as that makes a lot of sense for many shops.

What would be the correct way going forward of not displaying the root category view with all the products?

Previous issue that has been closed: https://github.com/PrestaShop/PrestaShop/issues/37040 

### Expected behavior

Same as it was before update, disabling root category disables the view but would still show child categories.

### Steps to reproduce

1. Have at least 1 subcategory
2. Disable root category

### PrestaShop version(s) where the bug happened

8.2.0

### If your bug is related to a module, specify its name and its version

ps_categorytree v3.0.0

**Comment by paulnoelcholot** [comment]
Hello @wwatu1337,

This problem seems to have already been fixed on version 3.0.1, can you do an update and try again?

Thanks for your feedback!

**Comment by wwatu1337** [comment]
@paulnoelcholot 

not really, if you disable the root cateogyr, none of the categories are shown, it just renders an empty box

**Comment by paulnoelcholot** [comment]
Hello @wwatu1337,

I reproduce the issue with PrestaShop version 8.2.x and 9.0.x, I'll add this to the backlog so it can be fixed.

Please be aware that some issues might take a very long time to be resolved. If this one is important to you and you cannot wait for it to be fixed on the project’s own time, we strongly suggest you consider [contacting a professional](https://www.prestashop-project.org/support/) to help you. 

If you fix the issue on your end, please [contribute it back to the project](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.

Thank you

8.2.x : 
https://www.loom.com/share/ef802be9420e441eace3974bebb244a3?sid=531648b7-dc38-4430-94d5-90e62fccca82

9.0.x : 
https://www.loom.com/share/2ae794269ac3496ea1ed43bab000f304?sid=c646b3b9-2568-454d-8d40-d36e6f368a89
